addons/portal/static/src/chatter/frontend/message_model_patch.js

```diff
@@ -0,0 +1,17 @@
+import { Message } from "@mail/core/common/message_model";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(Message.prototype, {
+    getReactParams(content) {
+        return {
+            ...super.getReactParams(...arguments),
+            ...this.thread.securityParams,
```

Comment:
For all of those, I'm wondering if it wouldn't be easier if the base code directly handled something like `...thread.rpcParams`, and we only override `rpcParams` method once to add the security params in portal.

Rather than having to provide a specific param method for each and to override each method individually.